include/userobjects/PICStudyBase.h

```diff
@@ -0,0 +1,52 @@
+//* This file is part of FENIX: Fusion Energy Integrated Multiphys-X,
+//* A multiphysics application for modeling plasma facing components
+//* https://github.com/idaholab/fenix
+//*
+//* FENIX is powered by the MOOSE Framework
+//* https://www.mooseframework.inl.gov
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "Function.h"
+#include "ClaimRays.h"
+#include "RayTracingStudy.h"
+#include "VelocityUpdaterBase.h"
```

Comment:
```suggestion

class VelocityUpdaterBase;
```

You can remove this and only "forward declare" it. All the compiler needs to know in this is that it is a class. When you go acutally call methods on this object (in the source file), you include it there. We put forward declarations after an extra newline after the includes.